.github/SUPPORT.md

```diff
@@ -1,26 +1,108 @@
-# Support
+# Node.js Support Guide
 
 Node.js contributors have limited availability to address general support
-questions. Please make sure you are using a [currently-supported version of
-Node.js](https://github.com/nodejs/Release#release-schedule).
+questions. Please make sure you are using a
+[currently-supported version of Node.js][1].
```

Comment:
We should probably align with the other Markdown document on the repo, which use the text of the link as reference
```suggestion
[currently-supported version of Node.js][].
```

Reply:
Because this PR references some links several times, I figured numbered references were easier (for diff size)